src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -1,28 +1,87 @@
 package seedu.address.logic.commands;
-/*
-import static java.util.Objects.requireNonNull;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.testutil.Assert.assertThrows;
 
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.function.Predicate;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.AddCommand.MESSAGE_DUPLICATE_JERSEY_NUMBER;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import javafx.collections.ObservableList;
-import seedu.address.commons.core.GuiSettings;
 import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
-import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.lineup.Lineup;
+import seedu.address.model.lineup.LineupName;
 import seedu.address.model.person.Person;
+import seedu.address.testutil.LineupBuilder;
 import seedu.address.testutil.PersonBuilder;
-*/
+
 public class AddCommandTest {
+    private static final Person VALID_PERSON = new PersonBuilder().withName("Joel")
+            .withEmail("joel@example.com").withPhone("1267912")
+            .withHeight("221").withJerseyNumber("83").withWeight("120")
+            .withTags("C").build();
+    private static final Person INVALID_PERSON = new PersonBuilder().withName("Alice Pauline").build();
+    private static final Person INVALID_PERSON_2 = new PersonBuilder()
+            .withName("Daniel Lee").withJerseyNumber("2").build();
+    private static final Lineup VALID_LINEUP = new Lineup(new LineupName("Dummy"));
+    private static final Lineup VALID_LINEUP_2 = new LineupBuilder().build();
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());;
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        model.addLineup(VALID_LINEUP);
+        expectedModel.addLineup(VALID_LINEUP);
+    }
+
+    @Test
+    public void execute_addPerson_success() throws CommandException {
+        expectedModel.addPerson(VALID_PERSON);
+        CommandResult commandResult = new AddCommand(VALID_PERSON).execute(model);
+        assertEquals(commandResult.getFeedbackToUser(),
+                String.format(AddCommand.MESSAGE_ADD_PERSON_SUCCESS, VALID_PERSON));
+    }
+
+    @Test
+    public void getExecute_invalidAddPerson_duplicatePerson() {
```

Comment:
Ah forgotten to tag you that I have included checks for duplicate person and duplicate jerseynumber. Nonetheless, the addition of lineup checking for add command is great!